stdlib/3/builtins.pyi

```diff
@@ -278,15 +278,15 @@ class str(Sequence[str]):
     def strip(self, chars: Optional[str] = None) -> str: ...
     def swapcase(self) -> str: ...
     def title(self) -> str: ...
-    def translate(self, table: Dict[int, Any]) -> str: ...
+    def translate(self, table: Union[Mapping[int, Union[int, str, None]], Sequence[Union[int, str, None]]]) -> str: ...
     def upper(self) -> str: ...
     def zfill(self, width: int) -> str: ...
     @staticmethod
     @overload
-    def maketrans(x: Union[Dict[int, Any], Dict[str, Any]]) -> Dict[int, Any]: ...
+    def maketrans(x: Mapping[Union[int, str], _T]) -> Dict[int, _T]: ...
     @staticmethod
     @overload
-    def maketrans(x: str, y: str, z: str = ...) -> Dict[int, Any]: ...
+    def maketrans(x: str, y: str, z: str = ...) -> Dict[int, Union[int, str, None]]: ...
```

Comment:
When would the return type include str? Looking at the C code for this branch it seems to only add ints and None.

Reply:
I didn't look into the code and just erred on the safe side to declare a return type that would for sure be understood by `str.translate` (since it's the same). Currently there is symmetry with these calls, but I can of course just change this to be more accurate.